src/components/Gauge.js

```diff
@@ -0,0 +1,263 @@
+const _isEqual = require('lodash/isEqual');
+const React = require('react');
+const Radium = require('radium');
+const d3 = require('d3');
+
+const StyleConstants = require('../constants/Style');
+
+const Gauge = React.createClass({
+  propTypes: {
+    activeOffset: React.PropTypes.number,
+    arcWidth: React.PropTypes.number,
+    baseArcColor: React.PropTypes.string,
+    chartTotal: React.PropTypes.number,
+    children: React.PropTypes.node,
+    colors: React.PropTypes.array,
+    data: React.PropTypes.array.isRequired,
+    dataPointColors: React.PropTypes.array,
+    dataPointRadius: React.PropTypes.number,
+    dataPoints: React.PropTypes.array,
+    defaultLabelText: React.PropTypes.string,
+    defaultLabelValue: React.PropTypes.string,
+    formatter: React.PropTypes.func,
+    height: React.PropTypes.number,
+    id: React.PropTypes.string,
+    opacity: React.PropTypes.number,
+    padAngle: React.PropTypes.number,
+    showBaseArc: React.PropTypes.bool,
+    showDataLabel: React.PropTypes.bool,
+    width: React.PropTypes.number
+  },
+
+  getDefaultProps () {
+    return {
+      activeOffset: 0,
+      arcWidth: 10,
+      baseArcColor: StyleConstants.Colors.BASE_ARC,
+      colors: [StyleConstants.Colors.PRIMARY].concat(d3.scale.category20().range()),
+      data: [],
+      dataPointColors: [StyleConstants.Colors.CHARCOAL].concat(d3.scale.category20b().range()),
+      dataPointRadius: 5,
+      dataPoints: [],
+      formatter (value) {
+        return value;
+      },
+      height: 150,
+      id: 'gauge',
+      opacity: 1,
+      padAngle: 0.02,
+      showBaseArc: true,
+      showDataLabel: true,
+      width: 150
+    };
+  },
+
+  getInitialState () {
+    return {};
+  },
```

Comment:
If initial state returns an empty object then there is no need to include the function.  You can just remove it.